WordPress/Tests/WP/I18nUnitTest.4.inc

```diff
@@ -0,0 +1,10 @@
+<?php
```

Comment:
There is no need for a new unit test case file. These tests could (and should) just have been added to the `I18nUnitTest.1.inc` file. Just add them to the end of the file just above the property reset.

Reply:
84f5173